persistence/remote/src/main/java/org/infinispan/persistence/remote/configuration/RemoteStoreConfiguration.java

```diff
@@ -52,13 +52,15 @@ public Object readAttributeValue(String enclosingElement, AttributeDefinition at
          .immutable().build();
    static final AttributeDefinition<String> REMOTE_CACHE_NAME = AttributeDefinition.builder("remoteCacheName", "").immutable().xmlName("cache").build();
 
+   static final AttributeDefinition<String> URI = AttributeDefinition.builder("uri", "").immutable().xmlName("uri").build();
```

Comment:
The default value should be `null` here, not an empty string.

Reply:
Setting `XmlName` is not required either, as the value is the same as the attribute name.